packages/nestjs/README.md

```diff
@@ -477,16 +455,6 @@ The following options are available for the BacktraceClientOptions passed when i
 | `breadcrumbs`                       | BacktraceBreadcrumbsSettings                        | See [Backtrace Breadcrumbs](#breadcrumbs)                                                                                                                                                                                                                                                                                                                                            |         | <ul><li>- [ ] </li></ul> |
 | `database`                          | BacktraceDatabaseSettings                           | See [Backtrace Database](#offline-database-support)                                                                                                                                                                                                                                                                                                                                  |         | <ul><li>- [ ] </li></ul> |
 
-#### BacktraceClient Methods
-
-| Name                                                                                                                                            | Return Type     | Description                                                                       |
-| ----------------------------------------------------------------------------------------------------------------------------------------------- | --------------- | --------------------------------------------------------------------------------- |
-| `addAttribute(attributes: Record<string, unknown>)`                                                                                             | void            | Add attributes to the BacktraceClient reports                                     |
-| `addAttachment(attachment: BacktraceAttachment)`                                                                                                | void            | Add an attachment to the BacktraceClient reports                                  |
-| `initialize(options: BacktraceClientOptions)`                                                                                                   | BacktraceClient | Initializes a new BacktraceClient (returns the same instance on subsequent calls) |
-| `builder(options: BacktraceClientOptions).build()`                                                                                              | BacktraceClient | (Advanced) Sets up a new BacktraceClient for reporting                            |
-| `send(data: BacktraceReport \| Error \| string, reportAttributes: Record<string, unknown> = {}, reportAttachments: BacktraceAttachment[] = [])` | Promise<void>   | Asynchronously sends error data to Backtrace                                      |
-
 #### BacktraceDatabase Methods
```

Comment:
my idea was to remove this section. I believe we do not need to maintain it here. what do you think?